modules/aws-emr-config/versions.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.36, !=4.0.0, !=4.1.0, !=4.2.0, !=4.3.0, !=4.4.0, !=4.5.0, !=4.6.0, !=4.7.0, !=4.8.0"
+      version = ">=4.0.0"
```

Comment:
I think we'd want to be consistent to say `>=4.9.0` to match the version at the root. If you use the root, the constraint would put you at a version >=4.9.0, but using this nested module directly wouldn't enforce that.